ui5.yaml

```diff
@@ -1,21 +1,22 @@
-specVersion: "2.0"
+specVersion: "3.0"
 metadata:
   name: ui5.typescript.helloworld
 type: application
 framework:
   name: OpenUI5
-  version: "1.104.0"
+  version: "1.112.0"
   libraries:
     - name: sap.m
     - name: sap.ui.core
     - name: sap.ui.unified
-    - name: themelib_sap_fiori_3
+    - name: themelib_sap_horizon
+builder:
+  customTasks:
+    - name: ui5-tooling-transpile-task
+      afterTask: replaceVersion
 server:
   customMiddleware:
-  - name: ui5-middleware-livereload
-    afterMiddleware: compression
-    configuration:
-      debug: true
-      extraExts: "xml,json,properties"
-      port: 35729
-      path: "webapp"
+    - name: ui5-tooling-transpile-middleware
+      afterMiddleware: compression
+    - name: ui5-middleware-livereload
+      afterMiddleware: compression
```

Comment:
look at the reduction of all tooling extensions - isn't that nice now how simple it is here? no config for the transpile and the livereload anymore